src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -183,6 +192,34 @@ internal override CommonAnonymousTypeManager CommonAnonymousTypeManager
         /// </summary>
         internal bool IsPeVerifyCompatEnabled => LanguageVersion < LanguageVersion.CSharp7_2 || Feature("peverify-compat") != null;
 
+        internal bool RunNullableWalker
```

Comment:
>RunNullableWalker [](start = 22, length = 17)

nit: name is misleading (this doesn't "run" anything). Consider `ShouldRunNullableWalker` or inversely `CanSkipNullableWalker`.